include/zephyr/drivers/firmware/scmi/power.h

```diff
@@ -16,6 +16,10 @@
 
 #define SCMI_POWER_STATE_SET_FLAGS_ASYNC BIT(0)
 
+/* SCMI power domain generic states */
+#define SCMI_POWER_STATE_GENERIC_ON    0x00000000U
+#define SCMI_POWER_STATE_GENERIC_OFF   0x40000000U
```

Comment:
Not sure the state value is unified definitions in SCMI protocol or could be SoC specific? we could define them at here if it is unified definitions crossing different vendor's SoC,  otherwise suggest to move them to be vendor's definitions, such as put them in subdirectory: nxp/power.h.